_posts/2017-09-30-mozilla.md

```diff
@@ -0,0 +1,58 @@
+
+Stay up to date and get notified when Firefox Quantum launches: https://www.mozilla.org/en-US/firefox/quantum/
+
+
+## What comes next in Firefox Quantum?
+
+With the release of Firefox Quantum we are not done yet. WebRender is a project that is [currently being worked on](https://mozillagfx.wordpress.com/category/wr-newsletter/). WebRender is written in Rust and uses parallelism to render websites.
+
+## Get involved and test Mozilla Quantum!
+
+The Mozilla community is organizing testing events for Firefox Quantum.
+
+We invite you to help us test websites from your area, head over to the
+[Firefox Quantum Sprint
+website](https://firefoxsprint.mozilla.community/) to find an event.
+
+## Open development
+
+The Mozilla teams bringing Rust to Firefox have been busy writing about what they bring to Firefox. We assembled a collection of the most interesting blog posts for you!
+
+## Browser internals, mostly Rust
+
+*  [https://hacks.mozilla.org/2017/05/quantum-up-close-what-is-a-browser-engine/](Quantum Up Close: What is a browser engine?)
+
+* [https://hacks.mozilla.org/2017/08/inside-a-super-fast-css-engine-quantum-css-aka-stylo/](Inside a super fast CSS engine: Quantum CSS (aka Stylo))
+
+* [https://hacks.mozilla.org/2017/06/an-inside-look-at-quantum-dom-scheduling/](An inside look at Quantum DOM Scheduling)
+
+## General Engineering blog posts
+
+* [https://blog.mozilla.org/firefox/quantum-performance-test/](Firefox Quantum is super fast, while still conserving memory)
+
+* [https://blog.mozilla.org/firefox/get-ready-firefox-quantum/](Get ready for Firefox Quantum)
+
+* [https://blog.mozilla.org/firefox/fast-responsive-reliable-browser-multi-tab-age/](Fast, Responsive, Reliable: A Browser For the Multi-Tab Age)
+
+* [https://hacks.mozilla.org/2017/09/firefox-quantum-developer-edition-fastest-firefox-ever/](Firefox Quantum Developer Edition: the fastest Firefox ever with Photon UI and better tooling)
+
+* [https://hacks.mozilla.org/2017/06/designing-for-performance-a-data-informed-approach-for-quantum-development/](Designing for performance: A data-informed approach for Quantum development)
+
+* [https://hacks.mozilla.org/2017/09/developer-edition-devtools-update-now-with-photon-ui/](Developer Edition Devtools Update: Now with Photon UI)
+
+## Interesting Newsletters to read
+
+* [https://ehsanakhgari.org/tag/quantumflow](Quantum Flow project newsletter)
```

Comment:
This list is not seperated by empty lines like the others. I would make the others like this.
However the style sould be consistent